pkg/proto/configuration/bb_worker/bb_worker.proto

```diff
@@ -299,6 +299,16 @@ message VirtualBuildDirectoryConfiguration {
   // bazel on Windows effectively assumes that the file system is case
   // insensitive, so make sure this is set to true when using the WinFSP.
   bool case_insensitive = 6;
+
+  // The ID of the user that owns directories contained in the build
+  // directory. This should typically be set to the user that is used by
+  // bb_runner to run actions.
+  uint32 build_directory_owner_user_id = 7;
```

Comment:
Ideally these options live inside RunnerConfiguration. The reason being that it's not uncommon to run a single worker and multiple runners. The latter having a distinct user ID each.